cf/test/test_CellMethod.py

```diff
@@ -137,6 +137,12 @@ def test_CellMethod_get_set_delete(self):
         self.assertTrue(cm1.axes == ('time',))
 
 
+    def test_CellMethod_intervals(self):
+        cm, = cf.CellMethod.create('lat: mean (interval: 1 hour)')
+
+        self.assertEquals('1 hour', str(cm.intervals[0]))
```

Comment:
```suggestion
        self.assertEqual('1 hour', str(cm.intervals[0]))
```

preferred since when I run this I see:

```
test_CellMethod_intervals (__main__.CellMethodTest) ... test_CellMethod.py:143: DeprecationWarning: Please use assertEqual instead.
  self.assertEquals('1 hour', str(cm.intervals[0]))
```